molior-deploy

```diff
@@ -615,21 +616,29 @@ umount_bootstrap()
   fi
 }
 
-include_deploy_conf()
+include_package()
```

Comment:
it breaks compatibility, all projects using include_deploy_conf would need to be changed, upgrades probably would not be possible due to renamed function 

-> create a new function for include_file